app/src/hooks/useOptions.ts

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios';
+import { useQuery } from 'react-query';
+
+import { Option } from '../types';
+
+// fetches options and return only choices
+const getOptions = async () => {
+  const { data } = await axios({
```

Comment:
Why not use the more "native" `fetch` instead of axios to make the requests? Any thoughts?
I'm asking it because every package we add to the project increases the bundle size.

Reply:
Oh, I'm just used to axios, but sure, I can rewrite this with fetch. Keeping bundle size as small as possible makes sense